desktop/package.json

```diff
@@ -33,23 +33,23 @@
   "license": "MIT",
   "private": true,
   "dependencies": {
-    "deep-diff": "^0.3.4",
-    "deep-equal": "^1.0.1",
+    "deep-diff": "0.3.4",
+    "deep-equal": "1.0.1",
     "framed-msgpack-rpc": "git://github.com/keybase/node-framed-msgpack-rpc#f3e1a3eb410e0bcde950bae14ceef7015c7315c8",
     "getenv": "0.6.0",
     "immutable": "3.8.1",
     "jsonfile": "2.3.1",
-    "lodash": "4.13.1",
-    "marked": "0.3.5",
-    "material-ui": "0.15.2",
-    "menubar": "5.0.0",
+    "lodash": "4.14.2",
+    "marked": "0.3.6",
```

Comment:
Where is marked used in the codebase? Some quick grepping didn't turn up anything for me.